test/integration/targets/nxos_aaa_server_host/tests/common/radius.yaml

```diff
@@ -130,6 +130,24 @@
 
   - assert: *false
 
+  - name: "Configure radius server with new type 7 encryption key"
+    nxos_aaa_server_host:
+      server_type: radius
+      address: 8.8.8.8
+      host_timeout: 25
+      auth_port: 2083
+      acct_port: 2084
+      encrypt_type: 7
+      key: helloback
+      provider: "{{ connection }}"
+      state: present
+    register: result
+
+  - assert:
+      that:
+        - "result.changed == true"
+        - "'key 7' in result.updates[0]"
```

Comment:
Will you please add idempotence test like https://github.com/ansible/ansible/blob/devel/test/integration/targets/nxos_aaa_server_host/tests/common/radius.yaml#L127?